eureka/src/main/resources/logback-access.xml

```diff
@@ -3,7 +3,7 @@
     <jmxConfigurator />
     <statusListener class="ch.qos.logback.core.status.OnConsoleStatusListener" />
     <property scope="local" name="accessLogPattern" value="%date{yyyy-MM-dd HH:mm:ss.SSS} traceid=&quot;%i{x-b3-traceid}&quot; spanid=&quot;%i{x-b3-spanid}&quot; parentspanid=&quot;%i{x-b3-parentspanid}&quot; method=&quot;%m&quot; url=&quot;%U&quot; query=&quot;%q&quot; protocol=&quot;%H&quot; statusCode=&quot;%s&quot; duration=&quot;%D&quot; bytesSent=&quot;%b&quot; remoteIp=&quot;%a&quot; remoteHost=&quot;%h&quot; localIp=&quot;%A&quot; localServer=&quot;%v&quot; localPort=&quot;%localPort&quot; userAgent=&quot;%i{User-Agent}&quot;" />
-    <property scope="local" name="fileName" value="${spring.application.home:-.}/logs/water-service.access.log" />
+    <property scope="local" name="fileName" value="${spring.application.home:-.}/logs/eureka.access.log" />
```

Comment:
Good catch, do you want to do this (with the other `logback-access` filename fixes) in a separate PR?

Reply:
Created #17